tests/aws/services/kms/test_kms.py

```diff
@@ -2006,6 +2006,22 @@ def test_derive_shared_secret(self, kms_create_key, aws_client, snapshot):
             )
         snapshot.match("response-invalid-public-key", e.value.response)
 
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(paths=["$..CurrentKeyMaterialId"])
+    def test_describe_with_alias_arn(self, kms_create_key, aws_client, snapshot):
+        snapshot.add_transformer(snapshot.transform.key_value("CurrentKeyMaterialId"))
+
+        alias_name = f"alias/{short_uid()}"
+        created_key = kms_create_key(Description="test - description")
+        snapshot.match("created-key", created_key)
```

Comment:
Is there a reason we capture the create key response here? I think we can remove the snapshot since we cover this usage in a lot of other tests already, wdyt?